pages/search.tsx

```diff
@@ -55,20 +55,31 @@ const ViewModeActionsContainer = styled.div`
   }
 `
 
-const UnstyledList = styled.ul`
-    margin: 0;
-    padding: 0;
-    list-style-type: none;
-  `
-
 const CheckboxContainer = styled.div`
   display: flex;
   flex-wrap: wrap;
-  grid-gap: 0.5rem;
+  grid-gap: 0.3rem;
+  
 `
 
 const SearchResultsList = styled(List)`
-  list-style: none;
+  list-style: none
+`
+
+const CommunityList = styled(List)`
+padding-inline-start: 0;
+font-weight: 500;
```

Comment:
You can remove lines 71-73 because they have no effect due to the label styles. I guess you have tried to add the styles in the Figma sketches here? I don't think we should care about them as they are not according to EDS anyway. Using EDS compact as you already have done is more than enough 👍

Reply:
Removed font styling :D